### ISSUE TITLE
User should be able to receive private messages only from favorite users/ or block users

### ISSUE THREAD

**Issue by lipsmega** [q2a/question2answer#589]
Im owner of a popular q2a brazilian website and this is the biggest problem with it. I think that
users should be able to receive private messages only from favorite users/ or block users that shouldnt send them messages. tnks!

**Comment by BurninLeo** [comment]
I agree that the message feature is an invitation for spam, as long as there's no limitation on who may sent messages to whom. Such a limitation may be based on connections as suggested by @lipsmega or on achievements/points (e.g., only one message per day for user with 200-500 points) etc.

On the other side, private messages are not a necessary featore for a q2a platform. I chose to disable it completely in my Q2A installation.

**Comment by lipsmega** [comment]
I know that private messages is not to important for q2a websites, but its a great feature that improves so much the communication through the community.
